react-native/android/src/main/jni/Application.mk

```diff
@@ -1,7 +1,7 @@
 APP_BUILD_SCRIPT := Android.mk
 
 APP_ABI := armeabi-v7a x86 x86_64 arm64-v8a
-APP_PLATFORM := android-9
+APP_PLATFORM := android-16
```

Comment:
there should be no need to bump this version

Reply:
You where right it works with the NDK ```21.0``` I move it back to ```android-9```.

Reply:
Yep, maybe we need to explicitly state this in the readme to be 21.0